### DISCUSSION TITLE
Updates for 2024 Zephyrus Devices

### DISCUSSION THREAD

**Discussion by mugiwaraboi** [seerge/g-helper#2044] [General]
Hey, I have gotten the new G14 2024. Will G Helper be updated to include the new functionalities of this device such as the slash lighting   , etc.

**Comment by seerge** [comment]
@mugiwaraboi hello,

G-Helper should already work on G14 2024. With exception of slash lightning indeed :)

To add support for slash lightning I need someone with the actual device to help me (i suspect it works very similarly to current anime-matrix in older G14, as it's probably just a single-line matrix de facto)

https://github.com/seerge/g-helper/discussions/2046

**Reply by mugiwaraboi** [reply]
Hey @seerge I have seen your new discussion. I'll try my best to do as instructed, and hope I can help! :)
